.github/workflows/InvalidationFlagger.yml

```diff
@@ -20,14 +21,29 @@ jobs:
     - uses: julia-actions/julia-invalidations@v1
       id: invs_master
     
-    - name: Report invalidation counts
+    - name: Print invalidation report
+      id: report
       run: |
         echo "Invalidations on master: ${{ steps.invs_master.outputs.total }} (${{ steps.invs_master.outputs.deps }} via deps)"
         echo "This branch: ${{ steps.invs_pr.outputs.total }} (${{ steps.invs_pr.outputs.deps }} via deps)"
-      shell: bash
-    - name: PR doesn't increase number of invalidations
-      run: |
-        if (( ${{ steps.invs_pr.outputs.total }} > ${{ steps.invs_master.outputs.total }} )); then
-            exit 1
-        fi
-      shell: bash
+      shell: bash  
+      
+    - name: Report increased invalidations (fail)
+      if: ${{ steps.invs_pr.outputs.total > steps.invs_master.outputs.total }}
+      uses: dflydev/check-runs-action@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        output: '{"summary":Invalidations on master: ${{ steps.invs_master.outputs.total }} (${{ steps.invs_master.outputs.deps }} via deps)\nThis branch: ${{ steps.invs_pr.outputs.total }} (${{ steps.invs_pr.outputs.deps }} via deps)"}'
+        id: report
+        name: "${{ steps.invs_pr.outputs.total }} (${{ steps.invs_master.outputs.deps }} via deps) vs. ${{ steps.invs_master.outputs.total }} on master"
+        conclusion: "failure"
```

Comment:
we could remove the `if` check and change the `conclusion` to:

```
        conclusion: ${{ steps.invs_pr.outputs.total > steps.invs_master.outputs.total }} ? "failure" : "success"
```

This reduces the code amounts a bit.

Reply:
Tried this and `${{ steps.invs_pr.outputs.total > steps.invs_master.outputs.total ? "failure" : "success" }}` but both failed to be parsed

Reply:
Oh my bad, it should be

`${{ steps.invs_pr.outputs.total }} > ${{ steps.invs_master.outputs.total }} ? "failure" : "success"`

each variable should be wrapped by `${{ ... }}`

Reply:
that still wouldn't parse unfortunately. I broke it up with the code duplication